english/dragon-slave/dragon-slave.md

```diff
@@ -0,0 +1,10 @@
+A wide-eyed beast looms over head. It is covered with thick plated scales that threaten death with every glint of the sun. It slowly inhales, its broad chest expanding across the sky, blotting your hope of survival.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.